README.md

```diff
@@ -279,7 +279,7 @@ If a `handler` is passed to the `customRetry` object the onus is on the client t
 Given example
 
 ```js
-  const customRetryLogic = (req, res, getDefaultRetry) => {
+   const customRetryLogic = (req, res, getDefaultRetry, err) => {
```

Comment:
Do you like this signature or `(req, res, err, getDefaultRetry)` better? The original PR hasn't been released yet so I think we can make a breaking change here if we want.